cpp/tests/table/row_operator_tests_utilities.hpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
Similarly.
```suggestion
 * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Reply:
This is really a brand new file. None of the declarations technically existed previously since the definitions were all self-contained.